docs/mobile-apps/automated-testing/espresso-xcuitest/espresso-capture.md

```diff
@@ -0,0 +1,39 @@
+---
+id: espresso-capture
+title: Espresso Screenshot Capture
+sidebar_label: Espresso Screenshot Capture
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The Espresso Screenshot Capture functionality allows you to take screenshots of errors to support the visual verification of your debugging process. These screenshots also help to identify any errors or bugs within your application.
+
+
+## Capturing Screenshots
+
+To take screenshots during an Espresso test run, follow the steps below. Make sure to handle exceptions in your test code to ensure a proper test run. This feature is supported on Android 4.3 and above.
+
+### To capture screenshots:
+
+1. Add the utility class to your project. See our Java example on [GitHub](https://github.com/saucelabs/my-demo-app-android/blob/2daaab68f6b75dcd78533dda7ac1715eec070f99/app/src/androidTest/java/com/saucelabs/mydemoapp/android/screenshot/SauceLabsCustomScreenshot.java).
```

Comment:
Can we make the life of the users easier by:
- showing the code here, this is an extra click and not easy to copy
- add a Kotlin example

Reply:
Consulted the case with @kristofmuhi . Kotlin is not mandatory for the first release and it is not a blocker for this MR.